lib/k8s/client.rb

```diff
@@ -68,7 +68,17 @@ def self.in_cluster_config(namespace: nil, **options)
     # @return [K8s::Client]
     def self.autoconfig(namespace: nil, **options)
       if ENV.values_at('KUBE_TOKEN', 'KUBE_CA', 'KUBE_SERVER').none? { |v| v.nil? || v.empty? }
-        configuration = K8s::Config.build(server: ENV['KUBE_SERVER'], ca: ENV['KUBE_CA'], auth_token: options[:auth_token] || ENV['KUBE_TOKEN'])
+        unless Base64.decode64(ENV['KUBE_CA']).match?(/CERTIFICATE/)
```

Comment:
Is this worth anything?

(The KUBE_CA must be base64, it's decoded and ran through various magics of OpenSSL in transport.rb)

It does not strict_decode64 because transport.rb doesn't either, maybe it could?